globals.go

```diff
@@ -476,7 +476,7 @@ func IsGitHubPatchRequester(requester string) bool {
 }
 
 // Permissions-related constants
-var AclCheckingIsEnabled = false
+var AclCheckingIsEnabled = (os.Getenv("ACL_ENABLED") == "true")
```

Comment:
put this in settings, and make it toggleable elsewhere=

Reply:
This feels a bit weird as a setting since this should always be on when the project is done. Putting it in settings also adds more boilerplate to tests, since they'll need to set the field in setup code